CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html),
 with the exception that 0.x versions can break between minor versions.
 
+## [Unreleased]
+### Added
+- Support `\Z` - anchor to the end of the text before any trailing newlines. (#148)
+- Support `\O` - any character including newlines. (#158)
+- The parser can now parse subroutine calls and relative backreferences. This is preparation for future work. Some new error variants have been added for features which can be parsed but are still otherwise unsupported.
```

Comment:
Maybe makes it a bit clearer:

```suggestion
- The parser can now parse subroutine calls and relative backreferences (but execution is still unsupported). This is preparation for future work. Some new error variants have been added for features which can be parsed but are still otherwise unsupported.
```